ui/app/models/kubernetes/config.js

```diff
@@ -3,7 +3,12 @@ import { withFormFields } from 'vault/decorators/model-form-fields';
 import { withModelValidations } from 'vault/decorators/model-validations';
 
 const validations = {
-  kubernetesHost: [{ type: 'presence', message: 'Kubernetes host is required' }],
+  kubernetesHost: [
+    {
+      validator: (model) => (model.disableLocalCaJwt && !model.kubernetesHost ? false : true),
```

Comment:
Is there a case where the config may be set manually using the CLI, and `disabled_local_ca_jwt` is true, but then the user wants to come to the UI to edit the config to use the local cluster?

Reply:
Yes I think that would be possible. The user would still be able to update the config and add a value for `kubernetes_host`.